manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "DuckDuckGo for Chrome",
```

Comment:
This change is optional, but I recommend it because once entering "extension mode" (`d` + `Space` in the omnibox), this setting specifies the [label](https://developer.chrome.com/extensions/omnibox) that is shown in the omnibox before every search query, and it should be both as succinct and as semantic as possible.

![](https://www.evernote.com/shard/s381/sh/82d62d26-16a0-449d-a3b8-e1bbd5e164ec/4cc6c485aeaa559e875dbf95e8adab89/res/9eee84c8-7667-4ca7-aefd-4ece3f446082/skitch.png)